src/workflows/report_rhel_add_data.conf

```diff
@@ -0,0 +1,27 @@
+EVENT=workflow_RHELAddDataLibreport analyzer=libreport
+# this is just a meta event which consists of other events
+# the list is defined in the xml file
+
+EVENT=workflow_RHELAddDataCCpp type=CCpp
+# this is just a meta event which consists of other events
+# the list is defined in the xml file
+
+EVENT=workflow_RHELAddDataPython type=Python component!=anaconda
+# this is just a meta event which consists of other events
+# the list is defined in the xml file
+
+EVENT=workflow_RHELAddDataKerneloops type=Kerneloops
+# this is just a meta event which consists of other events
+# the list is defined in the xml file
+
+EVENT=workflow_RHELAddDatavmcore type=vmcore
+# this is just a meta event which consists of other events
+# the list is defined in the xml file
+
+EVENT=workflow_RHELAddDataxorg type=xorg
+# this is just a meta event which consists of other events
+# the list is defined in the xml file
+
+EVENT=workflow_RHELAddDataJava type=Java
+# this is just a meta event which consists of other events
+# the list is defined in the xml file
```

Comment:
Isn't workflow_RHELAddDataJavaScript missing here?

Reply:
Good point, we should add this EVENT even into ``src/workflows/report_rhel.conf`` file.

Reply:
On the basis of this https://github.com/abrt/libreport/commit/0f146787a70387564282b8f27dd6baea78d7fee3 I started to think the workflow was not added into ``src/workflows/report_rhel.conf`` by purpose (added only to Fedora because of experimenting). @jfilak am I right or is the missing workflow in ``src/workflows/report_rhel.conf`` bug?

Reply:
@mhabrnal  is right. JavaScript support is not mature enough. I didn't want to it to be displayed in the configuration window.

The referenced commit contains a bug in `workflow_RHELJavaScript.xml.in` -> `<_name>Report to Fedora</_name>`.